Sources/Identity.swift

```diff
@@ -102,4 +137,41 @@ import Foundation
             createXDMSharedState(data: state.identityProperties.toXdmData(), event: event)
         }
     }
+
+    // MARK: GetUrlVariables helpers
+
+    /// Helper function to generate url variables in format acceptable by the AEP web SDK
+    static func generateURLVariablesPayload(configSharedState: [String: Any], identityProperties: IdentityProperties) -> String {
+        // append timestamp
+        var theIdString = appendParameterToUrlVariablesString(original: "", key: IdentityConstants.URLKeys.TIMESTAMP_KEY, value: String(Int(Date().timeIntervalSince1970)))
+
+        // append ecid
+        if let ecid = identityProperties.ecid {
+            theIdString = appendParameterToUrlVariablesString(original: theIdString, key: IdentityConstants.URLKeys.MARKETING_CLOUD_ID_KEY, value: ecid)
+        }
+
+        // append org id
+        if let orgId = configSharedState[IdentityConstants.ConfigurationKeys.EXPERIENCE_CLOUD_ORGID] as? String {
+            theIdString = appendParameterToUrlVariablesString(original: theIdString, key: IdentityConstants.URLKeys.MARKETING_CLOUD_ORG_ID, value: orgId)
+        }
+
+        // encode adobe_mc string and append to the url
+        let urlFragment = "\(IdentityConstants.URLKeys.PAYLOAD_KEY)=\(URLEncoder.encode(value: theIdString))"
+
+        return urlFragment
+    }
+
+    /// Helper function to append key value to the url variable string
```

Comment:
Add documentation for parameters and return value.
https://github.com/adobe/aepsdk-core-ios/blob/main/AEPIdentity/Sources/URLAppender.swift#L103